src/ts/src/NavBar.module.scss

```diff
@@ -0,0 +1,15 @@
+@import "common/styles/mixins.scss";
+
+.navbar {
+    background-color: var(--primary);
+    min-height: 50px;
+}
+
+.titleLink {
+    color: var(--white);
+
+    &:hover {
```

Comment:
can you remind me what the `&` does in scss?

Reply:
The `&` means this applies when an element has both the scope class and this class. So in this case, the stuff in the `&:hover { }` block will only apply to elements that have the `titleLink` *and* the `hover` class.